package.json

```diff
@@ -9,5 +9,8 @@
     "karma-jasmine": "^3.3.1",
     "karma-jasmine-html-reporter": "^1.5.4",
     "karma-spec-reporter": "^0.0.32"
+  },
+  "scripts": {
+    "test": "karma start --single-run --browsers ChromeHeadless"
```

Comment:
It'd probably be nice to add a couple additional browsers here, but starting with just Chrome SGTM.